.github/workflows/build-container.yml

```diff
@@ -448,9 +454,9 @@ jobs:
       - name: Check coredump existence
         if: success() || failure()
         id: check_coredumps
-        uses: andstor/file-existence-action@20b4d2e596410855db8f9ca21e96fbe18e12930b # v2, pinned to SHA for security reasons
+        uses: andstor/file-existence-action@20b4d2e596410855db8f9ca21e96fbe18e12930b  # v2, pinned to SHA for security reasons
         with:
-            files: "/tmp/coredump/core.*"
+          files: "/tmp/coredump/core.*"
```

Comment:
BTW @gbregman I recently used `systemd-coredump` and it's geat. Coredumps inside a container are automatically available to the host's `coredumpctl` command, and you can easily debug as if you were doing it inside:
```
CONTAINER_ID=#<get from docker ps>
SYSROOT=$(docker inspect -f $'{{.GraphDriver.Data.MergedDir}}' $CONTAINER_ID)

# Let's launch GDB
gdb -iex "set sysroot $SYSROOT" $SYSROOT/usr/local/bin/nvmf_tgt /tmp/core
```